pkg/asset/manifests/gcp/cluster.go

```diff
@@ -0,0 +1,138 @@
+package gcp
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/apparentlymart/go-cidr/cidr"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/ptr"
+	capg "sigs.k8s.io/cluster-api-provider-gcp/api/v1beta1"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/installconfig"
+	"github.com/openshift/installer/pkg/asset/manifests/capiutils"
+)
+
+func generateSubnetName(infraID, machineRole string) string {
```

Comment:
I've added a common function for this to https://github.com/openshift/installer/pull/7944/files#diff-4b12935beafe4ebc250e91c7b766a310041e8b4c49520eda4feff3503918e47fR113 and can remove this when that PR merges.